docs/key-value-storage.md

```diff
@@ -0,0 +1,95 @@
+Your Nimbella account has been provisioned with Key-Value Storage and Object Storage. In this document, we are going to cover Key-Value Storage.
+
+A Key-Value Storage acts as a database where you create keys and apply values to those keys.
+
+The `nim` command will show your current content.
+
+```
+# To show the list of key-values (may be empty initially)
+nim key-value list
+```
+
+This document divides into these parts:
+
+- Conceptual Overview
+- Using a Key-Value Storage inside actions using a Nimbella SDK
+  - The `nodejs` SDK
+  - The `php` SDK
+  - The `python` SDK
+- Common Redis commands
+
+### Conceptual Overview
+
+A Key-Value Storage behaves like a database where you can create keys, apply values to those keys, and then call on them to extract the data that you need.
+
+In this visual example, you can store user info, such as their City, State, and Country, and apply the correct values to them. This allows you to keep track of data and request the data when needed.
+
+![](https://live.staticflickr.com/65535/50653417258_e27306c044_k.jpg)
+
+### Using a Key-Value Storage inside actions using a Nimbella SDK
+
```

Comment:
We need a note here such as "Nimbella KV uses Redis as the underlying implantation. Redis is ...".
This is to transition the reader to understand why we talk about KV then they'll see "redis" next.